src/plugins/xerces/README.md

```diff
@@ -2,10 +2,11 @@
 - infos/author = e1528532 <e1528532@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/xml
+- infos/needs =
 - infos/placements = getstorage setstorage
 - infos/status = recommended experimental unittest
+- infos/metadata = elektraXercesOriginalRootName
```

Comment:
In `METADATA.ini` we define the meaning of meta data. There are no collisions, if another plugin chooses to use the same name, the other plugin has to adhere the same specifications and must use the meta data in an interchangable way.

So if another plugin has some sort of rootname, one can import/export between these plugins without loosing the rootname.

The name you have currently chosen (`xerces/rootname`) defines the metadata private to the plugin. So there can be no "collision", but also no interchange between plugins.